### PR TITLE
[EN-2012] Edit github action getting-started guide

### PR DESCRIPTION
**Pull request by JonathanLoscalzo**
## What's changed in the PR?
- Github Action docs

## Rationale
- This guide need changes, because it is using an old approach

## How'd to test?
- Docs are not tested

### CODE REVIEW

File: docs/web/docs/1-getting_started/7-github.mdx [hunk 1]
```diff
@@ -228,33 +239,35 @@ properties:
       version: '2012-10-17'
   role_name: iambic_github_app_lambda_execution
 ```
+2. You can then apply the template via `iambic apply resources/github/iam_role_lambda.yaml`
 
-You will need to replace the "NAME_OF_AWS_ACCOUNT_THAT_HAS_IambicHubRole" with the account name of the AWS account holding the IambicHubRole. In the event you change the secret names, you need to update those values as well.
+<!-- You will need to replace the "NAME_OF_AWS_ACCOUNT_THAT_HAS_IambicHubRole" with the account name of the AWS account holding the IambicHubRole. In the event you change the secret names, you need to update those values as well. -->
```
Comment (smoy) [inline docs/web/docs/1-getting_started/7-github.mdx:244]: Are we sure we can skip these, one of the problem is we do really need to find out the mgmt account name of the organization. for example: `aws_hub_account_name`. this is not static, it does depend on what the customer requirement is

**Review by smoy — CHANGES_REQUESTED** [review]
I think we need to still call out the mgmt account name

File: docs/web/docs/1-getting_started/7-github.mdx (hunk 1 above)
Comment (JonathanLoscalzo) [reply docs/web/docs/1-getting_started/7-github.mdx:244]: after call: `NAME_OF_AWS_ACCOUNT_THAT_HAS_IambicHubRole`is not a role name, is it an example of account name

File: docs/web/docs/1-getting_started/7-github.mdx [hunk 2]
```diff
@@ -171,22 +177,31 @@ By installing IAMbic GitHub App, the app can integrate the pull-request and merg
...
+1. Place the above IAMbic template to your `iambic-templates` repo under `resources/github/iam_role_lambda.yaml`
 
 ```yaml
 template_type: NOQ::AWS::IAM::Role
 included_accounts:
-  - aws_hub_account_name
+	# You will need to replace the "NAME_OF_AWS_ACCOUNT_THAT_HAS_IambicHubRole" 
+	# with the account name of the AWS account holding the IambicHubRole. 
+	# In the event you change the secret names, you need to update those values as well.
+  - aws_hub_account_name
```
Comment (smoy) [inline docs/web/docs/1-getting_started/7-github.mdx:204]: can we update this from `aws_hub_account_name` to `NAME_OF_AWS_ACCOUNT_THAT_HAS_IambicHubRole`

**Review by smoy — COMMENTED** [review]
one last comment

**Pull request merged**